egs/wsj/s5/steps/nnet3/xconfig_to_configs.py

```diff
@@ -156,11 +156,26 @@ def write_config_files(config_dir, all_layers):
 
     config_basename_to_header = get_config_headers()
 
+    input_pairs = []
```

Comment:
keep input-nodes of networks, by assuming them only exists in front of config(first layer). If not, this is buggy, but previous version is also problematic. Should fix latter.